package com.github.libgraviton.workerbase.auth.exception;

public class CannotProcessAuth extends Exception {

    public CannotProcessAuth(String message, Throwable cause) {
        super(message, cause);
    }

}
